src/common/ini_handler.c

```diff
@@ -2,9 +2,17 @@
 
 #include "ini_handler.h"
 #include "ff.h"
+#include "eeprom.h"
+#include "lwip/dhcp.h"
+#include "lwip/netifapi.h"
```

Comment:
we need to see if this is needed here. ini handler does only loading ini file!

Reply:
So where should I put eeprom setting and netif setting? when we load ini file, we need to set those values...
Should I create separate file for this?